task2/client/client.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+)
+
+func main() {
+	conn, err := net.Dial("tcp", ":8083")
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("connected to ", conn.RemoteAddr().String())
+	defer fmt.Println("disconnected from ", conn.RemoteAddr().String())
+	defer conn.Close()
+
+	for {
+		input := bufio.NewReader(os.Stdin)
+
+		fmt.Println("enter data to send")
+
+		data, err := input.ReadString('\n')
+
+		if err != nil {
+			break
+		}
+
+		check := strings.TrimSuffix(data, string('\n'))
+
+		if check == "EXIT" {
```

Comment:
What happens if I write `exit` ?

Reply:
`exit` is not a command for exit. if `exit` is input, so this string will be sent to server and server will process this string and will return `EXIT`.

Reply:
From homework description:
C exits by input `exit`.

I expect that when client input == `exit`, client should be stopped. Am I correct?

Reply:
u are right. my fault.